README.md

```diff
@@ -6,57 +6,91 @@ This is the GitHub repository of the [OGC Environmental Data Retrieval API Stand
 
 The **OGC API - Environmental Data Retrieval** candidate standard is part of the OGC API suite of standards. [OGC API standards](https://ogcapi.ogc.org) define modular API building blocks to spatially enable Web APIs in a consistent way. [OpenAPI](http://openapis.org) is used to define the reusable API building blocks.
 
-**The public comment period on the EDR API closed September 28th.** [Go here for more information.](https://www.ogc.org/standards/requests/215)
+### EDR API Query Patterns
+The EDR API allows a query, constructed of a spatio-temporal pattern, to retrieve data just for that pattern, from a data collection resource. The main patterns are:
```

Comment:
I Think this line should be changed to:

The EDR API allows a query, constructed of a spatio-temporal pattern, to retrieve data just for that pattern, from a data collection resource. **Each of these patterns is optional, but a compliant API must implement at least one.**
The main patterns are:

Reply:
Done